### ISSUE TITLE
[Bug] Updater fails to update correctly

### ISSUE THREAD

**Issue by svioletg** [svioletg/viMusBot#23]

**Comment by svioletg** [comment]
From attempting to update from 1.7.0 to 1.7.0b:

```
Checking...
Your local version number:
1.7.0
...does not match the current latest release of:
1.7.0b
Would you like to update now? (y/n) y
Retrieving: https://api.github.com/repos/svioletg/viMusBot/zipball/1.7.0b
Extracting...
Copying...
[Errno 13] Permission denied: 'svioletg-viMusBot-3e2978b/setup-wizard'
Skipping setup-wizard...
Cleaning up...
Done!
You are now on v1.7.0.
```

The setup-wizard also being skipped needs to be fixed as well.

**Comment by svioletg** [comment]
I had the script stop after downloading the zip, seems like it might actually not be downloading the right one after all?

**Comment by svioletg** [comment]
Closer inspection, I think the files are downloading correctly... but the version.txt included in the latest release of 1.7.0b hadn't been updated, so the updater just kept thinking the version was outdated. I think what this has told me is that I need to figure out some sort of way to automate the version number changing, because I almost always forget to update the file manually.

That being said, I'll update the release to the latest commit soon.
